spring-integration-core/src/test/java/org/springframework/integration/dsl/routers/RouterTests.java

```diff
@@ -622,10 +627,19 @@ public void testNestedScatterGatherSequenceTest() {
 						.setReplyChannel(replyChannel)
 						.build());
 
+		Message<?> wiretapMessage1 = scatterGatherWiretapChannel.receive(10000);
+		assertThat(wiretapMessage1).isNotNull();
+		MessageHeaders headers1 = wiretapMessage1.getHeaders();
+		Message<?> wiretapMessage2 = scatterGatherWiretapChannel.receive(10000);
+		assertThat(wiretapMessage2).isNotNull()
+				.extracting(Message::getHeaders)
+				.isEqualToComparingOnlyGivenFields(headers1, "correlationId", "gatherResultChannel", "sequenceSize", "sequenceNo");
```

Comment:
Please, be sure that you don't exceed `120` line length.

Reply:
added additional line break.